sdk/samples/iot/aziot_esp32/create_trusted_cert_header.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
```

Comment:
Same for these scripts.  Seems we are duplicating a lot of code, making it harder to keep everything up to date if something was to change.  It does look like there are some subtle additions to these scripts that might be beneficial in the others of the same name?

Reply:
Yes, I have plans to consolidate these scripts and other common code as next step.